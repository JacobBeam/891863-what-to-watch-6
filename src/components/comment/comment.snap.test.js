import React from 'react';
import {render} from '@testing-library/react';
import Comment from './comment';

const comment = {
  "id": 1,
  "user": {
    "id": 4,
    "name": `Test`
  },
  "rating": 0,
  "comment": `Test Test Test Test Test Test Test Test`,
  "date": `2019-05-08T14:13:56.569Z`
};

it(`Should Comment render correctly`, () => {
  const {container} = render(<Comment comment={comment}/>);
  expect(container).toMatchSnapshot();
});
