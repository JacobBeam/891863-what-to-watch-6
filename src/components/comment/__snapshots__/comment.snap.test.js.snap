// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Comment render correctly 1`] = `
<div>
  <div
    class="review"
  >
    <blockquote
      class="review__quote"
    >
      <p
        class="review__text"
      >
        Test Test Test Test Test Test Test Test
      </p>
      <footer
        class="review__details"
      >
        <cite
          class="review__author"
        >
          Test
        </cite>
        <time
          class="review__date"
          datetime="2019-5-3"
        >
          May 8, 2019
        </time>
      </footer>
    </blockquote>
    <div
      class="review__rating"
    >
      0
    </div>
  </div>
</div>
`;
